File: Jetpack/databinding/src/main/java/com/twp/databinding/MainActivity.java
package com.twp.databinding;

import android.os.Bundle;
import android.view.LayoutInflater;
import androidx.appcompat.app.AppCompatActivity;
import com.twp.databinding.databinding.ActivityMainBinding;
import com.twp.databinding.lisenter.ClickHandler;
import com.twp.databinding.mv.User;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityMainBinding dataBinding = ActivityMainBinding.bind(LayoutInflater.from(this).inflate(R.layout.activity_main,null));
        User user = new User("刘备","刘玄德");
        ClickHandler handler = new ClickHandler();

        dataBinding.setUser(user);
        dataBinding.setClickHelp(handler);

    }


}

File: UI/fullscreen/src/main/java/com/twp/fullscreen/MainActivity.kt
package com.twp.fullscreen

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        tvWindowFlag.setOnClickListener {
            startActivity(Intent(this, WindowFlagActivity::class.java))
        }

        btnDecoerView.setOnClickListener {

            startActivity(Intent(this, DecorViewSystemUiVisibilityActivity::class.java))

        }
        btnThemeFullScree.setOnClickListener {
            startActivity(Intent(this, FullScreenInThemeActivity::class.java))
        }
        btnOpenGuide.setOnClickListener {
            startActivity(Intent(this, GuideActivity::class.java))
        }
        btnFullPreview.setOnClickListener {
            startActivity(Intent(this, FullPreviewImage::class.java))
        }
        btnNotchCompat.setOnClickListener {
            startActivity(Intent(this@MainActivity, NotchCampactActivity::class.java))
        }
    }
}
